changelog/issue-3615.md

```diff
@@ -0,0 +1,22 @@
+audience: general
+level: major
+reference: issue 3615
+---
+RFC 165 has been implemented, allowing for greater administrator control over
+"public" endpoints. Previously these were guarded by no scopes and could be
+accessed by anyone with no way to limit this. In this release all
+unauthenticated API calls are now granted the scope `assume:anonymous`.
+Additionally, most previously unprotected endpoints are now guarded by at
+least one scope, to enable the following:
+
+* To maintain current behavior, some scopes will need to be granted to the `anonymous`role. Refer to
+`the [anonymous role section](https://docs.taskcluster.net/docs/manual/deploying/anonymous-role) in the docs.
```

Comment:
```suggestion
  the [anonymous role section](https://docs.taskcluster.net/docs/manual/deploying/anonymous-role) in the docs.
```
It looks like a rogue backtick has gotten in here.

Reply:
Good catch!